docs: fix broken tables in browser-support.md

leandrodiniz opened the pull request:
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?
<!-- Please check the one that applies to this PR using "x". -->

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:


## What is the current behavior?
<!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->

Issue Number: N/A


## What is the new behavior?


## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No


<!-- If this PR contains a breaking change, please describe the impact and migration path for existing applications below. -->


## Other information

mary-poppins commented:
You can preview 143b17a at https://pr41122-143b17a.ngbuilds.io/.

jelbourn commented:
@leandrodiniz it's not clear from your PR/commit description what's actually broken here; it looks like the changes are mostly formatting. Additionally, this change seems to have broken the link rendering in the "Optional browser features to polyfill" table (you can see in the preview link posted above)

mary-poppins commented:
You can preview 2e89f2a at https://pr41122-2e89f2a.ngbuilds.io/.

leandrodiniz commented:
@jelbourn actually it's mostly formatting changes but that impacts the way the tables are displayed on GitHub. You can check the differences here:

**master:**
https://github.com/angular/angular/blob/master/aio/content/guide/browser-support.md

**my branch:**
https://github.com/angular/angular/blob/2e89f2ac0f78651653824efd8cf4f36a74d4c274/aio/content/guide/browser-support.md

and I also fixed the link rendering in the 'polyfills' table that was broken.

mary-poppins commented:
You can preview be74fa2 at https://pr41122-be74fa2.ngbuilds.io/.

jelbourn commented:
Ah, I see. Can you squash your changes into a single commit with a message that clarifies that this addresses rendering _on GitHub_. Something like

```
docs: format browser-support tables for GitHub markdown

Format the HTML used in browser-support.md such that the tables
render correctly under GitHub markdown.
```

leandrodiniz commented:
Done. I squashed my changes into a single commit with a clearer message.

gkalpak commented on aio/content/guide/browser-support.md:
```diff
@@ -3,89 +3,41 @@
+    <td>IE</td>
+    <td>11</td>
   </tr>
  <tr>
```
While you're at it, you could fix the indentation here too 🙏 

```suggestion
  <tr>
```

gkalpak commented on aio/content/guide/browser-support.md:
```diff
@@ -157,220 +109,133 @@ If the polyfill you want is not already in `polyfills.ts` file, add the `import`
-
-      [ES2015](guide/browser-support#core-es6)
-
+    <a href="guide/browser-support#core-es6" class="code-anchor">ES2015</a>
```
```suggestion
      <a href="guide/browser-support#core-es6">ES2015</a>
```

gkalpak commented on aio/content/guide/browser-support.md:
```diff
@@ -157,220 +109,133 @@ If the polyfill you want is not already in `polyfills.ts` file, add the `import`
     <td>
-
-      [AnimationBuilder](api/animations/AnimationBuilder).
+      <a href="api/animations/AnimationBuilder" class="code-anchor">AnimationBuilder</a>
```
```suggestion
      <a href="api/animations/AnimationBuilder">AnimationBuilder</a>
```

gkalpak commented on aio/content/guide/browser-support.md:
```diff
@@ -157,220 +109,133 @@ If the polyfill you want is not already in `polyfills.ts` file, add the `import`
-
-      [Web Animations](guide/browser-support#web-animations)
-
+      <a href="guide/browser-support#web-animations" class="code-anchor">Web Animations</a>
```
```suggestion
      <a href="guide/browser-support#web-animations">Web Animations</a>
```

gkalpak commented on aio/content/guide/browser-support.md:
```diff
@@ -157,220 +109,133 @@ If the polyfill you want is not already in `polyfills.ts` file, add the `import`
     <td>
-
-       [NgClass](api/common/NgClass) on SVG elements
+      <a href="api/common/NgClass" class="code-anchor">NgClass</a> on SVG elements
```
```suggestion
      <a href="api/common/NgClass">NgClass</a> on SVG elements
```

gkalpak commented on aio/content/guide/browser-support.md:
```diff
@@ -157,220 +109,133 @@ If the polyfill you want is not already in `polyfills.ts` file, add the `import`
-
-      [classList](guide/browser-support#classlist)
-
+      <a href="guide/browser-support#classlist" class="code-anchor">classList</a>
```
```suggestion
      <a href="guide/browser-support#classlist">classList</a>
```

gkalpak commented on aio/content/guide/browser-support.md:
```diff
@@ -157,220 +109,133 @@ If the polyfill you want is not already in `polyfills.ts` file, add the `import`
-
-      [Router](guide/router) when using
-      [hash-based routing](guide/router#location-strategy)
+      <a href="guide/router" class="code-anchor">Router</a> when using <a href="guide/router#location-strategy" class="code-anchor">hash-based routing</a>
```
```suggestion
      <a href="guide/router">Router</a> when using <a href="guide/router#location-strategy">hash-based routing</a>
```

gkalpak commented on aio/content/guide/browser-support.md:
```diff
@@ -157,220 +109,133 @@ If the polyfill you want is not already in `polyfills.ts` file, add the `import`
-
-      [ES7/array](guide/browser-support#core-es7-array)
-
+      <a href="guide/browser-support#core-es7-array" class="code-anchor">ES7/array</a>
```
```suggestion
      <a href="guide/browser-support#core-es7-array">ES7/array</a>
```

gkalpak reviewed (changes requested):
Thx, @leandrodiniz! 

OOC, why were you trying to view the file on GitHub? (Just making sure there are no links we need to update, because this file is not intended to be viewed in the source code directly.)

leandrodiniz committed:
docs: format browser-support tables for GitHub markdown

Format the HTML used in browser-support.md such that the tables
render correctly under GitHub markdown.

gkalpak reviewed (approved):
LGTM :+1: Thx, @leandrodiniz!
(FYI, I have rebased the PR on master to resolve a conflict and allow CI to run.)

OOC, why were you trying to view the file on GitHub? (Just making sure there are no links we need to update, because this file is not intended to be viewed in the source code directly.)

mary-poppins commented:
You can preview e489f66 at https://pr41122-e489f66.ngbuilds.io/.

jelbourn reviewed (approved):
LGTM

Reviewed-for: docs-packaging-and-releasing
